composer.json

```diff
@@ -9,7 +9,7 @@
     },
     "minimum-stability": "stable",
     "require": {
-        "php": ">=5.4,<8.0-dev",
+        "php": "~5.6.0",
```

Comment:
No, quite on purpose actually. SURFnet doesn't run Step Up on 7.0. So it may work, it may not work we don't know, if you want it to work you use 5.6 😄